frontend/packages/console-dynamic-plugin-sdk/src/extensions/console-types.ts

```diff
@@ -316,17 +316,17 @@ export type ListPageHeaderProps = {
   badge?: React.ReactNode;
 };
 
-export type CreateWithPermissionsProps = {
-  createAccessReview?: {
-    groupVersionKind: GroupVersionKind;
-    namespace?: string;
-  };
+export type CreateAccessReview = {
+  groupVersionKind: GroupVersionKind;
+  namespace?: string;
 };
 
-export type ListPageCreateProps = CreateWithPermissionsProps & {
-  groupVersionKind: GroupVersionKind;
+export type CreateWithPermissionsProps = {
```

Comment:
We should be careful. If this isn't a public component, we shouldn't export its props in the API. Instead we could use 

```
ListPageCreateProps['createAccessReview']
```

below and not change any of the types here.